admins/README.md

```diff
@@ -0,0 +1,157 @@
+# Administrators
+
+## Table of Contents
+
+* [Components](#components)
+* [Architecture](#architecture)
+
+## Components
+
+The BonnyCI system is made up of the following components:
+
+  * [Bastion](#bastion)
```

Comment:
Just need to move this list back two spaces. The sub-sections should be able to stay indented two spaces from their parent.